lib/test.js

```diff
@@ -1834,6 +1836,101 @@ queue.push(function () {
   ]);
 });
 
+// Transcription tests
+const TRANSCRIPTION_EXAMPLE = {
+  id: 'transcription_id',
+  recordingId: 'recording_id',
+  error: null,
+  createdAt: 'timestamp',
+  updatedAt: 'timestamp',
+  _links: {
+    "self": "/calls/call_id/legs/leg_id/recordings/recording_id/transcriptions/transcription_id",
+    "file": "/calls/call_id/legs/leg_id/recordings/recording_id/transcriptions/transcription_id.txt"
+  }
+};
+
+// Transcription create
+queue.push(function () {
+  nock(VOICE_ENDPOINT)
+    .post('/calls/call_id/legs/leg_id/recordings/recording_id/transcriptions', '{"language":"language"}')
+    .reply(200, {
+      data: [
+        TRANSCRIPTION_EXAMPLE
+      ]
+    });
+
+  messagebird.transcriptions.create('call_id', 'leg_id', 'recording_id', 'language', function (err, data) {
+    doTest(err, 'transcriptions.create', [
+      ['type', data instanceof Object],
+      ['.id', typeof data.data[0].id === 'string'],
+      ['.recordingId', typeof data.data[0].recordingId === 'string'],
+      ['.error', data.data[0].error === null],
+      ['.createdAt', typeof data.data[0].createdAt === 'string'],
+      ['.updatedAt', typeof data.data[0].updatedAt === 'string'],
```

Comment:
Indeed it can be improved, and I'll change to assert the data as well.

Neither the equals nor the strict equals operator works on objects as they have different references. It can be done by creating a deep object comparison, like the underscore's `isEqual`, and refactor all tests to do such comparison.

Reply:
Alright, this is fine for now :)